### PR TITLE
Deprecate access via symbols

### PR DESCRIPTION
**Pull request by Meekohi**
* *No functional code changes*
* Add deprecation notice when using symbols
* Update documentation and tests to use strings
* Bump version, update changelog

See #32

### CODE REVIEW

File: lib/airrecord/table.rb [hunk 1]
```diff
@@ -8,10 +8,24 @@ module Airrecord
...
   # doesn't use the same column keys as everything else.
+  #
+  # 2018-11-01
+  # deprecate_symbols: long-term plan is to force everyone to use raw strings,
+  # to match the Airtable behavior. For now we'll just warn when using symbols
+  # with a deprecation notice.
+
   class Table
+    def deprecate_symbols
+      warn Kernel.caller.first + ": warning: Using symbols with airrecord is deprecated."
```
Comment (sirupsen) [inline lib/airrecord/table.rb]:
```suggestion
      self.class.deprecate_symbols
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 0.2.5
```
Comment (sirupsen) [inline CHANGELOG.md]: Let's start adding a `1.0.0 (unreleased)` as a collecting TODO, in this case killing symbols.

**Comment by Meekohi** [comment]
@sirupsen have another look see if it's good to go. 👍

**Comment by sirupsen** [comment]
Perfect. I've added you as a collaborator, I need your rubygems email. I'd like you to publish this version yourself (you can bump the version in master and push) to make sure someone else has access :)

**Comment by Meekohi** [comment]
Made a rubygems account with meekohi@gmail.com, let me know when it's setup and I'll give it a try.

**Comment by sirupsen** [comment]
```
$ gem owner airrecord -a meekohi@gmail.com
Owner added successfully.
Owners for gem: airrecord
- meekohi@gmail.com
- sirup@sirupsen.com
```

Done!

**Comment by Meekohi** [comment]
Hey I see someone pushed it up today -- thanks! Sorry for the delay!

**Comment by sirupsen** [comment]
@Meekohi do you want to follow up with a PR for 1.0.0 which removes support for symbols (but raises with a nice error message if one is passed)?

**Pull request merged**